app/helpers/application_helper.rb

```diff
@@ -1,6 +1,6 @@
 module ApplicationHelper
   def full_title(page_title)
-    base_title = 'hackEDU'
+    base_title = 'Hack Club'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.